infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/GeometryService.kt

```diff
@@ -322,7 +326,7 @@ class GeometryService @Autowired constructor(
         logger.serviceCall("getVerticalGeometryListing", "planId" to planId)
         val planHeader = getPlanHeader(planId)
         val alignments = geometryDao.fetchAlignments(planHeader.units, planId)
-        val geocodingContext = geocodingService.getGeocodingContext(OFFICIAL, planHeader.trackNumberId)
+        val geocodingContext = getLayoutGeocodingContextForPlanTrackNumber(planHeader.trackNumber)
```

Comment:
Vanhastaan näin, mutta miksikäs tämä oikeastaan haluaa geokoodata layoutin kontekstissa eikä suunnitelman itsensä? En yhtään muista onko tästä aikanaan sovittu jotain, mutta äkkiseltään vaikuttaa epäintuitiiviselta.

Reply:
Tutkaisenpa tuota. Nimesin metodin noin juuri siksi että se hyppisi silmille, mutta sitten se ei enää hyppinyt minun omille silmille siltikään...

Reply:
Tämä on jo vanha juttu, mutta dokumentoin omat ajatukseni tänne.

Eli mielestäni paikannuspohjan rataosoitteet ovat tässä oikea valinta, koska niiden mukaan on mahdollista yhdistää asioita isompaan kokonaisuuteen ja suunnitelmien tietojen mukaan ei aina ole mahdollista laskea osoitteita.

Jos tulevaisuudessa tulee tarve, suunnitelman sisäiset osoitteet voisi laskea ja ilmoittaa lisätietona.